### ISSUE TITLE
Seems out of sequence

### ISSUE THREAD

**Issue by curlywallst** [learn-co-curriculum/collaborating-objects-review#6]
I think this lab(video) would be better later.  Avi goes over a lot of stuff that is sort of out of left field for online students.  He goes pretty fast and is covering a lot of material that we haven't seen yet.  I realized this as I just hit the inheritance lessons and understood what he was talking about back in this video.   There are several other things as well that I assume I will learn later.  But this made it hard to get much out of this video given that he is moving fast and once you're lost it is pretty hard to follow.

**Comment by curiositypaths** [comment]
Hi there,

Thank you for the suggestion. We will review the ordering to ensure this video is placed in the right location.

Generally speaking, the videos are meant as material to augment the written curriculum and should be viewed as optional. After hearing feedback from students wanting more videos we have placed a few throughout the course. Because every student learns differently though, these videos are simply another way to deepen your understanding. If it doesn't work for you that's no problem. To better communicate this we will add a disclaimer explaining all of this to the lessons containing video resources. The written curriculum will independently cover all the required knowledge. Of course, we will continue to improve the videos and written material.

If, after reviewing the written curriculum, you still feel that there is a gap, feel free to reopen the issue.

As always, thanks for contributing! 💙

**Comment by curlywallst** [comment]
Hi Jason - I would hate anyone missing the videos - they are so good!!  I just meant that if they followed the relevant material you would get so much more out of them.  Maybe you could reference them again at the end of the topic and say something about how great it would be to look at the again if they already saw them.  Just a thought.  Thanks..  Best, Nancy
